net-nds/openldap/openldap2.4-2.4.48.recipe

```diff
@@ -95,8 +111,12 @@ INSTALL()
 	mkdir -p $dataDir/openldap
 	mv $settingsDir/openldap/ldap.conf.default $dataDir/openldap/
 
+	packageEntries devel \
+		$developDir \
+		$documentationDir/man/man3
```

Comment:
for the next time: I would prefer `$manDir/man3`.

Reply:
Further allong the line $documentationDir is removed :)

Reply:
`$manDir` is equivalent to `$documentationDir/man`, just shorter :)